htdocs/ticket/card.php

```diff
@@ -188,7 +188,7 @@
 			$object->email_from = $user->email;
 			$notifyTiers = GETPOST("notify_tiers_at_create", 'alpha');
 			$object->notify_tiers_at_create = empty($notifyTiers) ? 0 : 1;
-
+			$object->fk_user_assign = GETPOST("fk_user_assign", 'int');
 			$object->fk_project = $projectid;
 
 			$id = $object->create($user);
```

Comment:
Is the status of ticket that is created is set automatically to "Assigned" when $object->fk_user_assign is set to a user id ?

Reply:
it's done now